plugin/pkg/shoot/quotavalidator/admission.go

```diff
@@ -229,7 +229,7 @@ func (q *QuotaValidator) Validate(ctx context.Context, a admission.Attributes, o
 	}
 
 	// Admit Shoot lifetime changes
-	if lifetime, exists := shoot.Annotations[common.ShootExpirationTimestamp]; checkLifetime && exists && maxShootLifetime != nil {
+	if lifetime, exists := shoot.Annotations[common.ShootExpirationTimestampDeprecated]; checkLifetime && exists && maxShootLifetime != nil {
```

Comment:
Why is here no handling for the new/non-deprecated expiration timestamp annotation?

Reply:
I was thinking about some corner cases about this annotation but I guess it is fine to have the handling for for the new/deprecated annotation. Thanks!